src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1857,6 +1865,12 @@ def load_arguments(self, _):
             "disable_fips_image",
             action="store_true"
         )
+        c.argument(
+            'localdns_config',
```

Comment:
other than declare the new parameters in this file, please add them as parameters to the corresponding entry function, in your case, it should be aks_agentpool_update in custom.py

Reply:
Added them to aks_agentpool_update and aks_agentpool_add and named the parameter "localdns_config" to make it match up with this argument name. Will that be enough to correlate the two?

Reply:
yep, we'll verify the change by live test